tests/data/test_study/extract_config_development.py

```diff
@@ -0,0 +1,92 @@
+from kf_lib_data_ingest.common import constants
```

Comment:
I wanted to reuse/import extract_configs/simple_tsv_example1.py_in each of my `extract_config<app_mode>.py` extract configs, but I couldn't do it because I need extract_configs to be a python package with a `__init__.py` file. Extract breaks when I add the `__init__.py` file since its not a real extract config. We should prob add a check to skip over this file